### DISCUSSION TITLE
Alguien que sepa de Phaser y Enable3D?

### DISCUSSION THREAD

**Discussion by sergiolopezloya** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#244] [Q&A]
Hola compañeros, aprovecho el portal para solicitar apoyo de alguien que sepa usar Phaser con Enable3D. Estoy desarrollando un video juego 3D con esas herramientas pero estoy teniendo problemas para reutilizar librerías cargadas desde un archivo glb.

Si alguien sabe como hacer esto les agradecería que me orientara que no hay mucha documentación al respecto.

**Comment by doguedogue** [comment]
orale, se ve interesante tipo el Roblox no? o bueno, ni idea, seguro aquí debe haber alguien que lo conozca 🚀

**Reply by sergiolopezloya** [reply]
Es más bien tipo Unity, Roblox es un poco más amigable pero restringido. En Phaser puedes hacer juegos con JavaScript y por lo tanto ejecutarse en web o compilar para dispositivos móviles, que es lo que estamos haciendo

**Comment by Ingrzs** [comment]
Ya checaste en stack? Prueba ahi, pero busca en el sitio de ingles no en el de español.

**Reply by sergiolopezloya** [reply]
Ya chequé en todo Google, siempre busco documentación en inglés, hay mucho más contenido, pero creo que este detalle es muy específico

**Comment by sergiolopezloya** [comment]
Miren, les comparto una imagen del juego :)
![image](https://user-images.githubusercontent.com/19512542/156813703-621ebc88-27ba-43d1-89f7-09882718d6e2.png)

**Comment by sergiolopezloya** [comment]
Pues que creen que ya pude corregirlo, para variar eran muy pocas líneas de código para que quedara, ahora sí vuela la app, igual gracias por responder.

En pocos días estará publicada la app en la versión alpha, y estaremos incorporando una versión free 2 play, por si les interesa verla:
https://play.google.com/store/apps/details?id=com.mysp.app

Saludos
